telemetry/definitions/commonDefinitions.json

```diff
@@ -178,11 +193,21 @@
             "type": "int",
             "description": "The amount of time required for the build to complete (in seconds)."
         },
+        {
+            "name": "buildPayloadBytes",
+            "type": "int",
+            "description": "The uncompressed payload size in bytes of the source files in customer project context"
+        },
         {
             "name": "buildSystemVersion",
             "type": "string",
             "description": "The build system version on the user's machine"
         },
+        {
+            "name": "buildZipFileBytes",
```

Comment:
what about number of files?

Reply:
We does not have a field for `Number of files` and I do not think it adds a value as file can be of any size and number does not define our latency or test generation experience.
Correct me If I am missing something.